packages/itwinui-css/src/utils/visually-hidden/classes.scss

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './visually-hidden';
+
+.iui-visually-hidden {
```

Comment:
will be important for react to include this css in `Menu` where we use this class.